.drone.yml

```diff
@@ -23,6 +23,7 @@ steps:
   when:
     event:
       - push
+      - pr
```

Comment:
You shouldn't need this -- all pull requests contain a push. If you do want a separate one for pull requests though, you'll need `pull_request`, not `pr`.